css/grid.styl

```diff
@@ -0,0 +1,227 @@
+// GRID
+
+.container
+  margin: 0 auto
+  max-width: 2400px 
```

Comment:
well actually you are correct i didnt think about it properly. there will always be an explicit highest value for container from the media queries so this will never actually do anything